projects/cpython3/build.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -eu
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Workaround for distutils, which doesn't copy $CC except on OSX.
+export LDSHARED="$CC -shared"
+# Workaround for PGen, which doesn't use CFLAGS for linking
+export LDFLAGS="$CFLAGS"
+# _freeze_importlib, and presumably all other binaries (including the fuzzer),
+# trip the leak detection.
+export ASAN_OPTIONS="detect_leaks=0"
+
+# Build python to run the app.
+pushd cpython
+./configure --prefix=$OUT/ --with-ensurepip=no
```

Comment:
Hm. Will this copy everything into $OUT? 
$OUT needs to contain only the fuzzer binaries and their dependencies, nothing extra

Reply:
On our fuzzing infrastructure, $OUY is different from the one that the binaries are built in. Will this still work, or are there hardcoded paths that will break?

Reply:
I don't understand the question. It works if I run this command, for example:

    python infra/helper.py build_image cpython3 && python infra/helper.py build_fuzzers --sanitizer=address cpython3 && python infra/helper.py run_fuzzer cpython3 fuzz_builtin_unicode

Reply:
When we build, $OUT is something like /out

When we run this on our fuzzing infrastructure, $OUT is something completely different. Are there paths etc that depend on things being in /out ?

Reply:
I don't know Python's built process well enough to answer either way. Is there a way to test this locally?